app/frontend/src/components/designer/settings/FormMetadataSettings.vue

```diff
@@ -0,0 +1,110 @@
+<script setup>
+import { storeToRefs } from 'pinia';
+import { useI18n } from 'vue-i18n';
+import { onMounted, ref } from 'vue';
+import { useFormStore } from '~/store/form';
+
+const { t, locale } = useI18n({ useScope: 'global' });
+const { form, isRTL } = storeToRefs(useFormStore());
+const techdocsLink =
+  'https://developer.gov.bc.ca/docs/default/component/chefs-techdocs/Capabilities/Integrations/Form-Metadata/';
```

Comment:
Looks like a deploy is needed to pick up the change with the URL here.

Reply:
deployed.

Reply:
Can this backslash be removed?
![image](https://github.com/user-attachments/assets/20be9acf-8e5d-49c2-b93a-89ea9d235bdb)
It's probably a Backstage-ism, but we might have people trying:
![image](https://github.com/user-attachments/assets/c7d26211-0b65-4d1d-a811-f49b9090cf8a)

Reply:
Yup, can remove it.